client/src/components/Offer/OfferDetails.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styles from "./Offer.module.scss";
+import { getNiceSalary } from "./OfferHelper";
+
+type Props = {
+  position: string;
+  company: string;
+  tags?: string[];
+  date?: string | Date;
+  salary: {
+    from: number;
+    to: number;
+    currency: string;
+  };
+  location?: string;
+  isNew?: boolean;
+};
```

Comment:
If most of them are from the Offer props, you can export the Offer props there and just use them there. If just some of them, you could do something like:
```
import { OfferProps } from './xxx';

...

type Props = Pick<OfferProps, 'position' | 'company' | etc.>;

```

In such a way, it won't be hardcoded but type-safe when something will change in the Offer props.

Reply:
Still valid, but appraently in different file